common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/AbstractRootConcept.scala

```diff
@@ -18,45 +16,6 @@ object Concept {
   }
 }
 
-// We're not extending this yet, as we don't actually want it to be part of
-// the Display model as yet before we've started testing, but in future it
-// might extend AbstractConcept
-case class InstantRange(from: Instant,
-                        to: Instant,
-                        label: String = "",
-                        inferred: Boolean = false) {
-
-  def withInferred(inferred: Boolean): InstantRange =
-    InstantRange(from, to, label, inferred)
-
-  // TODO: is label necessary? appears to always be same as Period.label
```

Comment:
You've lost this comment and the one above it.

Reply:
Have put the first comment back in, but left the second out (it was me when first coming to the code base, and don't think it's an issue now that I have had some more time understanding how we are modelling the data etc)